dev/provider_packages/PROVIDER_INDEX_TEMPLATE.rst.jinja2

```diff
@@ -55,6 +55,22 @@ for the minimum Airflow version supported) via
 Requirements
 ------------
 
+The minimum Apache Airflow version supported by this provider package is ``{{ MIN_AIRFLOW_VERSION }}``.
```

Comment:
cc: @eladkal  This line will be added to the docs of all providers in requirements. I think it's worth explicitly stating the min airflow version here, prominently as first requirement (Regardless if the package is preinstalled or not)